spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/source/TestIcebergSourceTablesBase.java

```diff
@@ -1290,6 +1295,12 @@ public void testUnpartitionedPartitionsTable() {
             .set("position_delete_file_count", 0)
             .set("equality_delete_record_count", 0L)
             .set("equality_delete_file_count", 0)
+            .set(
+                "total_data_size_in_bytes",
+                StreamSupport.stream(
+                        table.currentSnapshot().addedDataFiles(table.io()).spliterator(), false)
+                    .mapToLong(DataFile::fileSizeInBytes)
+                    .sum())
```

Comment:
probably worth extract a variable instead of inline the computation. 

also I saw you added coverage for unpartitioned table only, shall we also add one for partitioned table to make sure it s data size in bytes match for each partition?

Reply:
@dramaticlly Thank you for your feedback. 

Yes, we should add tests for partitioned table. I was able to do it for `testPartitionsTable` but not `testPartitionsTableDeleteStats` and `testPartitionsTableLastUpdatedSnapshot`.

Will dig into it more today.

Reply:
@dramaticlly I think I fixed `testPartitionsTableLastUpdatedSnapshot`, please take a look, thanks!